webapp/courses/static/courses/script.js

```diff
@@ -181,20 +181,41 @@ function show_term_description(span_slct, div_slct) {
         desc_div = $("#term-div-not-found");
     }
 
-    let pos = span.offset();
-    let height = span.height();
-    let arrow_height = 9;
+    let arrow_height = 10;
+    let arrow_width = 8;
+    let offset = span.offset();
+    let span_height = span.height();
+    let desc_div_height = desc_div.height();
+    let left_offset = offset.left;
+    let top_offset = offset.top + span_height + arrow_height
 
     var desc_content_div = desc_div.children("div.term-desc-contents");
     var desc_scrollable_div = desc_div.find("div.term-desc-scrollable");
     
-    if (desc_div.height() + "px" === desc_content_div.css('max-height')) {
+    if (desc_div_height + "px" === desc_content_div.css('max-height')) {
         desc_scrollable_div.slimScroll({
             height: desc_content_div.css('max-height')
         });
     }
     desc_div.css({"display" : "block"}); //This works in Jquery3 unlike .show()
```

Comment:
This line should probably be just before the currently last line (where the offset is set) to minimize the possibility of visibly teleporting popups.